tests/unit/Term/AliasGroupListTest.php

```diff
@@ -162,11 +162,11 @@ public function testGivenSetLanguage_removeByLanguageRemovesIt() {
 
 	/**
 	 * @dataProvider invalidLanguageCodeProvider
-	 * @expectedException InvalidArgumentException
 	 */
-	public function testGivenInvalidLanguageCode_removeByLanguageThrowsException( $languageCode ) {
-		$list = new AliasGroupList();
+	public function testGivenInvalidLanguageCode_removeByLanguageIsNoOp( $languageCode ) {
+		$list = new AliasGroupList( array( new AliasGroup( 'en', array( 'foo' ) ) ) );
 		$list->removeByLanguage( $languageCode );
+		$this->assertFalse( $list->isEmpty() );
```

Comment:
Can also be assertCount, an assertion checking for hasGroupForLanguage, an equals check with a clone or a second object, … I prefer to keep such code as simple as possible. 1. Setup non-empty list. 2. Trigger remove. 3. Should still be non-empty. Even assertTrue( true ) would be valid here, because this is a no-op test.